docs/REST_API-Draft/issues.md

```diff
@@ -0,0 +1,112 @@
+## Rzeczy do dodania:
+### Do serwisu autoryzacyjnego
+1.  Pobranie listy monitorów dla danego użytkownika:
+    * Monitor zawiera: adres, port, nazwę, jakieś id i nazwę użytkownika, który go utworzył (albo jego ID)
+    * na wejściu jwt
+    * na wyjściu lista monitorów
+1. Użytkownik może dodać monitor
+    * na wejściu jwt i monitor
+1. Użytkownik może usunąć monitor
+    * na wejściu jwt i monitor (lub id monitora)
+
+## Propozycje
+### Komunikcja Sensor -> Monitor
```

Comment:
Zgodnie z komentarzem prowadzącego to będzie realizowane inaczej(takiego maila dostałem od @kassqaa )